types/index.d.ts

```diff
@@ -2205,7 +2207,16 @@ export declare namespace Knex {
     queryContext(context: any): ColumnBuilder;
     after(columnName: string): ColumnBuilder;
     first(): ColumnBuilder;
+    check(checkPredicate: string, bindings?: any, constraintName?: string): ColumnBuilder;
```

Comment:
are bindings really `any`? Aren't they closer to `Record<string, any>`? Or that is not necessarily the case?

Reply:
I  will change it on other check function. It's seems that raw check on column make no sense (useful only on table) so this function need to be deleted.